src/frontend/scss/templates/courses/plugins/_licence_plugin.scss

```diff
@@ -20,8 +20,13 @@ $richie-licence-plugin-description-fontcolor: $gray40 !default;
   }
 
   &__wrapper {
+    margin-left: 10px;
     @include media-breakpoint-up($richie-licence-plugin-breakpoint) {
-      @include sv-flex(1, 0, calc(100% - #{$richie-licence-plugin-logo-width}));
+      @include sv-flex(
+        1,
+        0,
+        calc(100% - #{$richie-licence-plugin-logo-width + 10})
```

Comment:
This `10` comes from the margin above right? It should then be set in a variable and reused

Reply:
@mbenadda my problem is that the logo width is expressed in px so I get the error:

> "Incompatible units: 'rem' and 'px'."

Reply:
So I left px as discussed.